apps/sap-commerce-cloud/app-actions/README.md

```diff
@@ -1 +1,13 @@
 # SAP App Actions
+
+## Available scripts
```

Comment:
Maybe add a section "Actions" with links to the 3 actions and descriptions (which can be pulled from the app manifest). Vercel has an example of what I'm thinking here: https://github.com/contentful/apps/blob/master/apps/vercel/app-actions/README.md#actions

Reply:
Thanks for that example, great thing to include.